Solutions/MicrosoftDefenderForEndpoint/Hunting Queries/MDE_Usage.yaml

```diff
@@ -49,5 +51,4 @@ entityMappings:
         columnName: AlgorithmCustomEntity
       - identifier: Value
         columnName: FileHashCustomEntity
```

Comment:
can we update this as well to newer entity style?

Reply:
Done